modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -121,10 +121,13 @@ const renderContextToolbar = (spec: ContextToolbarSpec): ContextToolbarRenderRes
 
         AlloyEvents.run<ForwardSlideEvent>(forwardSlideEvent, (comp, se) => {
           InlineView.getContent(comp).each((oldContents) => {
+            const isIframe = SugarNode.isTag('iframe');
+            const activeElement = Focus.active(SugarShadowDom.getRootNode(comp.element));
+            const hasSelectionInside = activeElement.exists((e) => !isIframe(e) && e.dom.contains(se.event.currentSelection));
             stack.set(stack.get().concat([
               {
                 bar: oldContents,
-                focus: Focus.active(SugarShadowDom.getRootNode(comp.element))
+                focus: hasSelectionInside ? Focus.active(SugarElement.fromDom(se.event.currentSelection)) : activeElement
```

Comment:
Is there any reason to not just use `se.event.currentSelection` all the time?

Reply:
using always `se.event.currentSelection` would makes ['TINY-11748: Launch button should be able to navigate to a context toolbars and navigateback and retain toolbar focus'](https://github.com/tinymce/tinymce/blob/main/modules/tinymce/src/themes/silver/test/ts/browser/editor/contexttoolbar/ContextToolbarTest.ts#L173) fail, this is because in that case the `se.event.currentSelection` is the `link` but the `activeElement` (`Focus.active(SugarShadowDom.getRootNode(comp.element))`) is the button in the context toolbar, so using always `se.event.currentSelection` would mean not to preserve the focus on the context toolbar's buttons